### ISSUE TITLE
week day titles and header title shadows are hard coded.

### ISSUE THREAD

**Issue by aaronce** [Appgutta/AGCalendar#26]
Hi Chris, this is more of an issue with Kal  - but the shadows and font colours for some titles are hard coded in KalView.m "addSubviewsToHeaderView". Is this something you wish to work on at some stage? or would it be easier to ignore alter individually and recompile based on the project being worked on?

**Comment by ChrisRM** [comment]
The titles are currently localized, meaning it'll follow the phones language setting. If you want more dynamic titles, I could add some functionality for it, I just don't see the use case here.

**Comment by aaronce** [comment]
it's just when I am customising the UI png files some of the titles are a fixed colour and shadow, like I mentioned it's more of a Kal issue.. I can change manually it's okay :)

**Comment by ChrisRM** [comment]
Ah, Im in the work of making the module more customizable, so I'll drop this one in. I would love if you sent me a screenshot of the new UI you made. Maybe I'll add some different themes aswell.

**Comment by aaronce** [comment]
okay, will post one up as soon as I can.

**Comment by aaronce** [comment]
Hi Chris, here is the screenshot of altered UI, no huge modifications in layout just some colour changes.

edit: didn't realise the screenshot was so large sorry...

<img src="https://f.cloud.github.com/assets/914231/194706/0b8be6f0-7fba-11e2-9fde-8e47697aaaad.jpg" width="320">

By Chris: Just scaled down the image a bit :-)

**Comment by ChrisRM** [comment]
Nice job, I'll be releasing an experimental branch later tonight (CET+1). The new branch supports the following methods:
- `showTable` - Show(true) or hide(false) tableView (Defaults to true)
- `theme` - Yeah, style everything! See example below

``` javascript
var calendarView = Ti.Calendar.createView({
    top:0,
    editable: true,
    showTable: false, // Do not show event table (Default: true)
    theme: {
        tileView: {
            backgroundImage: {
                normal: /* Image */,
                selected: /* Image */,
                adjacent: /* Image */,
                today: /* Image */
            },
            textColor: {
                normal: /* Color */,
                today: /* Color */,
                selected: /* Color */,
                adjacent: /* Color */
            }
        },
        monthView: {
            backgroundImage:  /* Image */
        },
        mainView: {
            gridBackgroundImage:  /* Image */,
            gridDropShadowImage:  /* Image */,
            leftArrowImage:  /* Image */,
            rightArrowImage:  /* Image */,
            titleTextColor: /* Color */,
            titleShadowColor: /* Color */,
            titleShadowOffset: /* CGSize, format: {x:0,y:0} */,
            weekdayTextColor: /* Color */,
            weekdayShadowColor: /* Color */,
            weekdayShadowOffset: /* CGSize, format: {x:0,y:0} */
        },
        gridView: {
            backgroundImage: /* Image */,
            backgroundColor: /* Color */
        }
    },
    color:"white"
});
```

**Comment by aaronce** [comment]
nice, looks interesting, I'll have to give a it try.

**Comment by ChrisRM** [comment]
Fixed in version 2.0, will be merged in version 1.3
